index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>
+    <script src="./main.js" defer></script>
+    <script src="./profiles.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Gen Mex Concert 2023 Amazon Originals</title>
+</head>
+<body>
+    <div class="social-media-desktop display-none">
+        <div class="first-container-desktop">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <p>English</p>
+        <p>My Page</p>
+        <p>Logout</p>
+        </div>
+    </div>
+    <div class="nav-bar display-none">
+        <div class="nav-bar-container">
+        <a href="./index.html" class="logo"><img src="images/genmexlogo.jpg" alt="Gen Mex Concert Logo"></a>
+        <nav class="nav-menu">
+        <ul class="nav-options">
+            <li class="nav-link"><a class="link-menu" href="./about.html">About</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Program</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Join</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">Sponsor</a></li>
+            <li class="nav-link"><a class="link-menu" href="#">News</a></li>
+            <li class="nav-link"><a id="gmcampaign" class="link-menu" href="#">GM Campaign</a></li>
+        </ul>
+        </nav>
+        </div>
+    </div>
+    <header class="principal-info">
+        <div class="mobile-bar display-none">
+            <nav class="mobile-navmenu">
+            <ul class="mobile-nav-options">
+                <li class="mobile-nav-link"><a class="link-menu" href="./about.html">About</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Program</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Join</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">Sponsor</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">News</a></li>
+                <li class="mobile-nav-link"><a class="link-menu" href="#">GM Campaign</a></li>
+                <li class="mobile-nav-link"><i class="fa-solid fa-xmark"></i></li>
+            </ul>
+            </nav>
```

Comment:
- [OPTIONAL] I think it would be better if the `x` button is set in the top right of your nav bar menu :+1: because it would improve the User experience.
- [OPTIONAL] Also, kindly make sure that your nav bar covers the screen and the user can not access other sections when it is open :+1: 

**Your design when you open the nave bar**

![Screenshot from 2023-02-16 08-48-23](https://user-images.githubusercontent.com/105117832/219383062-ae1af7bc-596d-4c74-b824-de8e0daebc68.png)